proxy/hdrs/MIME.cc

```diff
@@ -3193,6 +3202,34 @@ mime_parse_uint(const char *buf, const char *end)
   }
 }
 
+uint64_t
```

Comment:
I think it would be better to put mime_parse_uint and mime_parse_uint64 into a template function and have the return value be an argument that is a reference.  I didn't see any difference in the functions besides the return type.

Example:
```
template<typename T>
void mime_parse_unsigned(const char *buf, const char *end, T &x)
```